### DISCUSSION TITLE
Suggestions

### DISCUSSION THREAD

**Discussion by vertigo220** [spacedriveapp/spacedrive#2747] [Feature Request]
I could be wrong, but this project seems to be primarily focused on the underlying purpose while spending very little time on actually developing a good file manager. Perhaps it's only intended to be very basic in that regard, but that would be a shame, as it seems to have a lot of potential there. And I feel that pulling everything together into one manager has limited usefulness if that manager isn't good at, well, managing.

There are multiple things I've found very quickly in trying it out that limit its usefulness as a file manager, i.e. that keep me from using it as my default:

- This was already mentioned and an issue created for it, but sorting folders first (and  perhaps optionally sorting folders mixed with files).
- The lack of a "My Computer"/"This PC" screen showing all the drives.
- A true list view (the current "list" view is really a "details" view).
- Remembering the list/grid setting (and columns/sorting/etc) for folders so they automatically return to that view when revisited (currently changing it in one folder changes it everywhere, causing it to have to be constantly changed back and forth; also, ideally should allow for creating rules, e.g. always show folders containing certain file types or folders matching a pattern (e.g. "D:\\.*\(") in a specific view).
- Clicking the "Explorer Settings" button at the top (should probably be named something like "View Settings" as that's all it is, and the name suggests it's more than that) and clicking the "Double click action" dropdown then clicking out of it closes the settings pop-up, even though clicking in that same space without the dropdown activated does not.
- The tabs aren't very visible (somewhat difficult to see and to see the borders).
- No multiple panes.
- The tree collapses every time a new tab is opened.
- It's impossible to grab the scroll bar with the mouse to fast scroll.
- Standard keyboard shortcuts don't work, e.g. Ctrl+Tab/Ctrl+Shift+Tab to cycle tabs.
- Files can't be dragged into a folder in spacedrive from another app.
- It's slow (opening a folder, switching tabs, even hitting home/end to scroll to the top/bottom of a folder are all noticeably slower than every other file manager).
- Even after focusing a tab so Home/End will work, when Alt-Tabbing away and back, the tab is no longer focused so they will work, and you have to click within it to focus it again.
- Typing to search doesn't work.
- Hyphens are sorted at the beginning, where I prefer hyphens to be ignored in sorting (perhaps an option to specify characters to be ignored).
- No option for setting default location when opening new tab.
- No ability to duplicate/rename/lock tabs.
- Copying path text should be quicker/easier/more obvious.
- No preview pane.
- No default right-click context menu (can't access typical workflow items).

I'm sure I could go on, but I'm not sure what the intent is for this project regarding how much of a file manager it wants to be, whether it's meant to be a fully-fledged one or just the bare necessities to accomplish its core mission of grouping files together from different sources. So there's my feedback so far, hopefully it's useful in giving ideas to improve it.

**Comment by jgarza9788** [comment]
my two cents.
As of now i'm using OneCommander (on windows), and gnome's default file manager on my linux.
I might switch to this one once it's done, because i see so much potential and i won't need to have two different file managers.
things that i'm hoping for ... 
* a little faster when saving configs and adding new folders to the side.
* a split-view, which is really handy when moving files.
* a column view (but only because i'm used to it existing)

maybe integrate with Everything (search tool)... this way it can import what i am indexing and either index the same folders, or just use the same index as everything.

**Reply by Be00wulf** [reply]
❤️

**Reply by jgarza9788** [reply]
I've also noticed some .spacedrive files get made, it was old be nice if uninstalling cleared these out.
